gcc/rust/lex/rust-token.cc

```diff
@@ -180,13 +229,13 @@ Token::as_string () const
       switch (get_id ())
 	{
 	case STRING_LITERAL:
-	  return "\"" + get_str () + "\"";
+	  return "\"" + escape_special_chars (get_str (), true) + "\"";
```

Comment:
tiny style thing, no need to do anything but if you have multiple flags passed to a function like this i would tend to write it as:

```escape_special_chars (get_str(), true /*in_string_flag*/)```

Reply:
Just makes it a little more readable without having to look at the function declaration you're calling

Reply:
Actually I don't know why I kept it like this, even better than a comment I would have used an enum.

Reply:
Rebased and improved.